components/PaywallModal.vue

```diff
@@ -0,0 +1,308 @@
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_1') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_2') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_3') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_4') }}</span>
+              </li>
+              <li class="flex items-start">
+                <UIcon
+                  name="i-material-symbols-check"
+                  class="mt-1 mr-2 text-green-500"
+                />
+                <span>{{ $t('pricing_page_feature_5') }}</span>
+              </li>
+            </ul>
+          </div>
+
+          <!-- Price Select -->
+          <div class="flex flex-col w-full mt-12">
+            <UIcon
+              v-if="props.dismissible"
+              name="i-material-symbols-close"
+              class="
+                hidden sm:block absolute top-4 right-4
+                text-black cursor-pointer hover:text-gray-600
+                transition-colors duration-200
+              "
+              size="24"
+              @click="handleOnClose"
+            />
+            <div class="flex flex-col gap-4">
+              <!-- Yearly plan -->
+              <label
+                class="
+                  relative flex justify-between items-center
+                  border-2 rounded-2xl px-4 py-4 cursor-pointer
+                  transition-all duration-200 ease-in-out
+                  hover:shadow-lg hover:border-gray-400
+                "
+                :class="selectedPlan === 'yearly' ? 'border-black' : 'border-gray-200'"
+              >
+                <div
+                  v-if="selectedPlan === 'yearly'"
+                  class="absolute -top-3 left-1/6 -translate-x-1/2 bg-black text-[#A6F5EA] text-xs font-semibold px-3 py-1 rounded-lg"
+                >
+                  {{ $t('pricing_page_yearly_discount', { discount: discountPercent }) }}
+                </div>
+
+                <div class="flex items-center">
+                  <div class="w-6 h-6 flex-shrink-0 mr-4">
+                    <div
+                      class="w-full h-full rounded-full border flex items-center justify-center bg-black"
+                      :class="selectedPlan === 'yearly' ? 'bg-black' : 'bg-white border-gray-300'"
```

Comment:
use array of classes

Reply:
[🎨 Refine UI style formatting](https://github.com/likecoin/3ook-com/pull/57/commits/ebc6d70152b905cd19f5c2c4e7feedb95f77cb1e)

Reply:
[🎨 Refine UI style formatting](https://github.com/likecoin/3ook-com/pull/57/commits/ebc6d70152b905cd19f5c2c4e7feedb95f77cb1e)